2010-07-28  Julian Brown  <julian@codesourcery.com>

	Backport from FSF mainline:

	gcc/
	* config/arm/thumb2.md (*thumb2_movdf_soft_insn): Fix alternatives
	for pool ranges.

 2010-07-26  Julian Brown  <julian@codesourcery.com>
 
 	Merge from Sourcery G++ 4.4:

=== modified file 'gcc/config/arm/thumb2.md'
--- old/gcc/config/arm/thumb2.md	2010-08-13 10:30:35 +0000
+++ new/gcc/config/arm/thumb2.md	2010-08-13 16:00:58 +0000
@@ -319,8 +319,8 @@
   "
   [(set_attr "length" "8,12,16,8,8")
    (set_attr "type" "*,*,*,load2,store2")
-   (set_attr "pool_range" "1020")
-   (set_attr "neg_pool_range" "0")]
+   (set_attr "pool_range" "*,*,*,1020,*")
+   (set_attr "neg_pool_range" "*,*,*,0,*")]
 )
 
 (define_insn "*thumb2_cmpsi_shiftsi"

